systemtest/src/test/java/io/strimzi/systemtest/bridge/HttpBridgeST.java

```diff
@@ -229,13 +226,9 @@ void testDiscoveryAnnotation() {
     void testScaleBridgeToZero() {
         String bridgeName = "scaling-bridge";
 
-        KafkaBridgeResource.kafkaBridge(bridgeName, KafkaResources.plainBootstrapAddress(CLUSTER_NAME), 1)
-            .editMetadata()
-                .addToLabels("type", "kafka-bridge")
-            .endMetadata()
-            .done();
+        KafkaBridgeResource.kafkaBridge(bridgeName, KafkaResources.plainBootstrapAddress(CLUSTER_NAME), 1).done();
 
-        List<String> bridgePods = kubeClient().listPodNames("type", "kafka-bridge");
+        List<String> bridgePods = kubeClient().listPodNames("type", KafkaBridge.RESOURCE_KIND);
```

Comment:
Isto to je spravne?
    public static final String RESOURCE_KIND = "KafkaBridge"; != ("type", "kafka-bridge")

Reply:
discussed offline, misunderstanding